src/components/DataFields/MarkdownEditor.tsx

```diff
@@ -22,6 +30,7 @@ interface Props extends FormControlProps {
   control: Control<any, any>;
   required?: boolean;
   disabled?: boolean;
+  max?: number;
```

Comment:
[nitpick] could we rename for clarity?

```suggestion
  maxLength?: number;
```

or

```suggestion
maxTextLength?: number;
```